rbac/api/cross_access/serializer.py

```diff
@@ -106,27 +108,59 @@ def get_roles(self, obj):
         serialized_roles = [RoleSerializer(role).data for role in obj.roles.all()]
         return serialized_roles
 
+    def throw_validation_error(self, source, message):
```

Comment:
Maybe we can make it a util for sharing, there is one for the CrossAccountRequestViewSet too: https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/api/cross_access/view.py#L201

Reply:
Done 👍